LayoutTests/inspector/dom/showGridOverlay.html

```diff
@@ -7,6 +7,14 @@
 {
     let suite = InspectorTest.createAsyncSuite("DOM.showGridOverlay");
 
+    const allEnabledOverlayOptions = {
```

Comment:
i wonder if we should do this for the flex overlay too?

Reply:
Can't hurt.